test/parallel/test-fs-write-file.js

```diff
@@ -95,3 +96,82 @@ fs.open(filename4, 'w+', common.mustSucceed((fd) => {
 
   process.nextTick(() => controller.abort());
 }
+
+{
```

Comment:
It would be better to separate the tests for the new functionality into a separate isolated test file.